File: map-engine-opengl2/src/jp/co/yahoo/map/codezine_sample/MainActivity.java
package jp.co.yahoo.map.codezine_sample;

import java.io.InputStream;
import java.io.IOException;

import com.example.codezine_sample.R;

import android.os.Bundle;
import android.app.Activity;
import android.content.res.AssetManager;
import android.view.Menu;

public class MainActivity extends Activity {
	private MapView mMapView ;							//MapView
	private static final String FILE_PATH = "01.txt";	//地図データ
	
	@Override
	protected void onCreate(Bundle savedInstanceState) {
		super.onCreate(savedInstanceState);
		
		//元からあるassetsフォルダに入れたtextファイルを読み込んでMapViewにセットします。
		try{
			AssetManager assets = getAssets();
			InputStream is = assets.open(FILE_PATH);
			
			//描画領域の作成
			mMapView = new MapView(this,is);
			setContentView(mMapView);
		}catch(IOException e) {
		}
	}
	
	@Override
	public boolean onCreateOptionsMenu(Menu menu) {
		getMenuInflater().inflate(R.menu.main, menu);
		return true;
	}
	
	@Override
	public void onDestroy(){
		if(mMapView != null) {
			mMapView.release();
			mMapView = null;
		}
		super.onDestroy();
	}
}

File: MapEngineOpenGL2/MapEngineOpenGL2/MainActivity.cs
using System;
using Android.App;
using Android.Content;
using Android.Runtime;
using Android.Views;
using Android.Widget;
using Android.OS;
using Android.Content.Res;
using Java.IO;

namespace MapEngineOpenGL2
{
	[Activity (Label = "MapEngineOpenGL2", MainLauncher = true)]
	public class MainActivity : Activity
	{
		private MapView mMapView ;                                                        //MapView
		private static string FILE_PATH = "01.txt";        //地図データ

		protected override void OnCreate(Bundle savedInstanceState) {
			base.OnCreate(savedInstanceState);

			//元からあるassetsフォルダに入れたtextファイルを読み込んでMapViewにセットします。
			try {
				var istream = Assets.Open(FILE_PATH);

				//描画領域の作成
				mMapView = new MapView(this, istream);
				SetContentView(mMapView);
			} catch(IOException e) {
			}
		}

		public override bool OnCreateOptionsMenu(IMenu menu) {
			MenuInflater.Inflate (Resource.Menu.main, menu);
			return true;
		}

		protected override void OnDestroy(){
			if(mMapView != null) {
				mMapView.Release();
				mMapView = null;
			}
			base.OnDestroy();
		}
	}
}


